test/lib/jdk/test/lib/process/ProcessTools.java

```diff
@@ -444,10 +444,11 @@ private static List<String> addTestThreadFactoryArgs(String testThreadFactoryNam
     /**
      * Create ProcessBuilder using the java launcher from the jdk to be tested.
      *
+     * @param isLimited Defines if jtreg options and test thread factory should be added
      * @param command Arguments to pass to the java command.
      * @return The ProcessBuilder instance representing the java command.
      */
-    private static ProcessBuilder createJavaProcessBuilder(String... command) {
+    private static ProcessBuilder createJavaProcessBuilder(boolean isLimited, String... command) {
```

Comment:
addThreadFactory might be a more appropriate name
However, Stefan has a more SOLID suggestion